<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StoreDoc</class>
 <widget class="QWidget" name="StoreDoc">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>893</width>
    <height>683</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget_4" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="btnSaveDoc">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnDraftDoc">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/draft.png</normaloff>:/images/draft.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnPrintDoc">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/printer.png</normaloff>:/images/printer.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QToolButton" name="btnRemoveDoc">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../Resort/res.qrc">
          <normaloff>:/images/garbage.png</normaloff>:/images/garbage.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>24</width>
          <height>24</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>761</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QWidget" name="wCommon" native="true">
        <layout class="QGridLayout" name="gridLayout_2">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item row="0" column="1">
          <widget class="EDateEdit" name="deDate"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="lbStoreout">
           <property name="text">
            <string>Store out</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="lbStorein">
           <property name="text">
            <string>Store in</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="EQLineEdit" name="leDocNumber">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="6">
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="2">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Number</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="EQLineEdit" name="leAction">
           <property name="EnableHiddenText" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Action</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Date</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Comments</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1" colspan="3">
          <widget class="EQLineEdit" name="leComments"/>
         </item>
         <item row="2" column="1">
          <widget class="EQLineEdit" name="leStore">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="EnableHiddenText" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="EQLineEdit" name="leStoreout">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="EnableHiddenText" stdset="0">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Payment</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="EQComboBox" name="cbPayment">
           <property name="Cache" stdset="0">
            <number>108</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="tabCommon">
         <attribute name="title">
          <string>Common</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout">
          <item row="2" column="5">
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Invoice date</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="EDateEdit" name="deInvoiceDate">
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="EQLineEdit" name="leInvoiceNo">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25552</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Invoice number</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>330</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Partner code</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="EQLineEdit" name="lePartnerCode">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>100</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="EQLineEdit" name="lePartnerName">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>0</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Partner name</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tabMaterials">
         <attribute name="title">
          <string>Materials</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QWidget" name="wGoodsOps" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <widget class="QToolButton" name="btnAddMaterial">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../Resort/res.qrc">
                 <normaloff>:/images/add.png</normaloff>:/images/add.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnAddMaterialByScan">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../Resort/res.qrc">
                 <normaloff>:/images/barcode.png</normaloff>:/images/barcode.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnRemoveMaterial">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../Resort/res.qrc">
                 <normaloff>:/images/cancel.png</normaloff>:/images/cancel.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="btnNewMaterial">
               <property name="text">
                <string>...</string>
               </property>
               <property name="icon">
                <iconset resource="../Resort/res.qrc">
                 <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>761</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <widget class="EQTableWidget" name="tblGoods">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="selectionMode">
             <enum>QAbstractItemView::SingleSelection</enum>
            </property>
            <property name="selectionBehavior">
             <enum>QAbstractItemView::SelectRows</enum>
            </property>
            <column>
             <property name="text">
              <string>RowId</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Goods Code</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Goods Name</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Qty</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Unit</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Price</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Total</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_5" native="true">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item>
              <spacer name="horizontalSpacer_5">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>712</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Total</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="EQLineEdit" name="leTotal">
               <property name="maximumSize">
                <size>
                 <width>100</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="readOnly">
                <bool>true</bool>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Cashdesk</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="WCashEntry" name="wAcc" native="true"/>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>EQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">eqlineedit.h</header>
   <slots>
    <signal>customButtonClicked(bool)</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>EDateEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">edateedit.h</header>
  </customwidget>
  <customwidget>
   <class>EQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">eqtablewidget.h</header>
  </customwidget>
  <customwidget>
   <class>EQComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">eqcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>WCashEntry</class>
   <extends>QWidget</extends>
   <header location="global">wcashentry.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resort/res.qrc"/>
 </resources>
 <connections/>
</ui>
